src/server/game/DataStores/DBCEnums.h

```diff
@@ -664,6 +664,62 @@ enum class ChrCustomizationReqFlag : int32
 
 DEFINE_ENUM_FLAG(ChrCustomizationReqFlag);
 
+enum CurrencyConsts
+{
+    CURRENCY_TYPE_ANCIENT_MANA    = 1155,
+    CURRENCY_TYPE_AZERITE         = 1553,
+
+    CURRENCY_MAX_CAP_ANCIENT_MANA = 2000
+};
+
+enum CurrencyTypesFlags : uint32
+{
+    CurrencyTradable                           = 0x00000001, // NYI
+    CurrencyAppearsInLootWindow                = 0x00000002, // NYI
+    CurrencyComputedWeeklyMaximum              = 0x00000004, // NYI
+    Currency_100_Scaler                        = 0x00000008,
+    CurrencyNoLowLevelDrop                     = 0x00000010, // NYI
+    CurrencyIgnoreMaxQtyOnLoad                 = 0x00000020,
+    CurrencyLogOnWorldChange                   = 0x00000040, // NYI
+    CurrencyTrackQuantity                      = 0x00000080,
+    CurrencyResetTrackedQuantity               = 0x00000100, // NYI
+    CurrencyUpdateVersionIgnoreMax             = 0x00000200,
+    CurrencySuppressChatMessageOnVersionChange = 0x00000400,
+    CurrencySingleDropInLoot                   = 0x00000800, // NYI
+    CurrencyHasWeeklyCatchup                   = 0x00001000, // NYI
+    CurrencyDoNotCompressChat                  = 0x00002000, // NYI
+    CurrencyDoNotLogAcquisitionToBi            = 0x00004000, // NYI
+    CurrencyNoRaidDrop                         = 0x00008000, // NYI
+    CurrencyNotPersistent                      = 0x00010000, // NYI
+    CurrencyDeprecated                         = 0x00020000, // NYI
+    CurrencyDynamicMaximum                     = 0x00040000,
+    CurrencySuppressChatMessages               = 0x00080000,
+    CurrencyDoNotToast                         = 0x00100000, // NYI
+    CurrencyDestroyExtraOnLoot                 = 0x00200000, // NYI
+    CurrencyDontShowTotalInTooltip             = 0x00400000, // NYI
+    CurrencyDontCoalesceInLootWindow           = 0x00800000, // NYI
+    CurrencyAccountWide                        = 0x01000000, // NYI
+    CurrencyAllowOverflowMailer                = 0x02000000, // NYI
+    CurrencyHideAsReward                       = 0x04000000, // NYI
+    CurrencyHasWarmodeBonus                    = 0x08000000, // NYI
+    CurrencyIsAllianceOnly                     = 0x10000000,
+    CurrencyIsHordeOnly                        = 0x20000000,
+    CurrencyLimitWarmodeBonusOncePerTooltip    = 0x40000000, // NYI
+    DeprecatedCurrencyFlag                     = 0x80000000
+};
+
+DEFINE_ENUM_FLAG(CurrencyTypesFlags);
+
+enum CurrencyTypesFlagsB : uint32
```

Comment:
```suggestion
enum class CurrencyTypesFlagsB : uint32
```
Same as CurrencyTypesFlags